#!/usr/bin/env python
PACKAGE = "masterslave"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("rosRate", int_t, 0, "ros rate", 1000, 20, 1000)
gen.add("gripperVelocity", double_t, 0, "velocity of the gripper", 0.1, 0.01, 100)
gen.add("Mode", str_t, 0, "mode", "Robot")
gen.add("safetyHeight", double_t, 0, "safety height", 0.05, 0, 2000)
gen.add("apertureLimit", int_t, 0, "aperture of the frustum", 60, 45, 80)
gen.add("start", bool_t, 0, "start flag", False)

exit(gen.generate(PACKAGE, "masterslave", "masterslave"))
